### PR TITLE
documentation update

### LINKED ISSUE #2: Add link to nice video from Sourceforge
www.youtube.com/embed/6ZqcJ6OZ598?rel=0

### LINKED ISSUE #493: Clean developer notes
https://github.com/pytrip/pytripgui/blob/master/developer_notes.rst

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #2 
Fixes #493

### CODE REVIEW

File: docs/technical.rst [hunk 1]
```diff
@@ -91,10 +96,37 @@ Features
...
+  * pytripgui is extracted to a separate project (https://github.com/pytrip/pytripgui)
+  * Leszek Grzanka joins developer team
+
+* 8.05.2018 pytripgui is migrated from wxwidgets to Qt5 framework (https://github.com/pytrip/pytripgui/commit/cb27fc909d132ce5f7a5e0be5df2dbbfd64e6c1d)
+
+  * GUI shifts completely from python 2.x to python 3.x
+
+* 04.2019 Lukasz Jelen joins developer team, introducing MVC architecture in the project
+
+* 06.2021 Arkadiusz Cwikla, Joanna Fortuna, Michal Krawczyk and Mateusz Lawczyk join project (part of bachelor thesis at AGH University)
```
Comment (rozchlastywacz) [inline docs/technical.rst:126]: Mateusz Łaszczyk

**Review by rozchlastywacz — APPROVED** [review]

File: docs/technical.rst (hunk 1 above)
Comment (rozchlastywacz) [reply docs/technical.rst:126]: you could also add info about our faculty and major if you want to be more precise

File: docs/technical.rst [hunk 2]
```diff
@@ -91,10 +96,37 @@ Features
...
+* 2014-2016 pytrip code (including GUI) is publicly available as a SourceForge project (https://sourceforge.net/projects/pytrip/)
+
+  *  Jakob Toftegaard issued several commits, Toke Printz included some fixes (https://github.com/pytrip/pytrip/commit/d6dedb8b5e309f33e06fb766542345064348e7e0)
+
+* 28.08.2016 pytrip (including GUI) is migrated to GIT at Github repository (https://github.com/pytrip/pytrip)
+
+  * pytripgui is extracted to a separate project (https://github.com/pytrip/pytripgui)
+  * Leszek Grzanka joins developer team
+
+* 8.05.2018 pytripgui is migrated from wxwidgets to Qt5 framework (https://github.com/pytrip/pytripgui/commit/cb27fc909d132ce5f7a5e0be5df2dbbfd64e6c1d)
```
Comment (nbassler) [inline docs/technical.rst:120]: 8.05 -> 08.05

Comment (grzanka) [reply docs/technical.rst]: done in 044de41

File: docs/technical.rst (hunk 1 above)
Comment (grzanka) [reply docs/technical.rst]: done in 044de41

File: AUTHORS.rst [hunk 3]
```diff
@@ -10,4 +10,12 @@ Development Lead
...
-None yet. Why not be the first?
+* Jakob Toftegaard
+* Niels Bassler
+* Toke Printz
+* Leszek Grzanka
+* Lukasz Jelen
+* Arkadiusz Cwikla
+* Joanna Fortuna
+* Michal Krawczyk
+* Mateusz Łaszczyk
```
Comment (rozchlastywacz) [inline AUTHORS.rst:21]: If you haven`t used polish letters in other names before, why did you used them in `Łaszczyk` name?

Comment (grzanka) [reply AUTHORS.rst:21]: My fault

**Pull request merged**